os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -89,7 +89,9 @@
 #ifdef CONFIG_APP_BINARY_SEPARATION
 #include <tinyara/binfmt/elf.h>
 #endif
-
+#ifdef CONFIG_SECURITY_LEVEL
```

Comment:
The CONFIG_SECURITY_LEVEL is correct.
because It is included for only to print security level.
```
#ifdef CONFIG_SECURITY_LEVEL
	lldbg("security level: %d\n", get_security_level());
#endif
```